### PR TITLE
Remove intersection observer as not needed due to site being static

### PR DESCRIPTION
**Pull request by agubler**
## Description

As the site is static we can remove the intersection observer.

### Code

This PR touches:

- [ ] Content
- [ ] Content Pipeline
- [x] Frontend
- [ ] Infrastructure

### Tests

- [ ] Tests are included?

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-40gvw5jmw.now.sh

File: src/home/GetGoing.tsx [hunk 1]
```diff
@@ -28,20 +23,20 @@ export default factory(function GetGoing({ middleware: { theme, intersection } }
...
-						<span classes={[themedCss.commandTwo, play ? themedCss.commandTwoAnimation : null]}>
+						<span classes={[themedCss.commandTwo, themedCss.commandTwoAnimation]}>
 							dojo create app --name hello-world
 						</span>
-						<span classes={[play ? themedCss.blinkTwo : null]}>|</span>
+						<span classes={[themedCss.blinkTwo]}>|</span>
 					</div>
 				</Card>
 				<div classes={[themedCss.codeContainer]}>
-					<Card dark classes={{ 'dojo.io/Card': { root: [play ? themedCss.code : themedCss.hide] } }}>
```
Comment (KaneFreeman) [inline src/home/GetGoing.tsx]: Should probably remove the `.hide` class completely from the css too.

Comment (agubler) [reply src/home/GetGoing.tsx]: @KaneFreeman good catch, done!

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-3jxtstlfe.now.sh

**Pull request merged**